dashboard/client/src/pages/job/JobRow.tsx

```diff
@@ -56,6 +57,29 @@ export const JobRow = ({
     }
   })();
 
+  const logsLink = (() => {
+    let link: string | undefined = undefined;
+    if (driver_agent_http_address) {
+      link = `/log/${encodeURIComponent(
+        `${driver_agent_http_address}/logs`,
+      )}?fileName=${type === "DRIVER" ? job_id : `driver-${submission_id}`}`;
+    } else if (driver_info && ipLogMap[driver_info.node_ip_address]) {
+      link = `/log/${encodeURIComponent(
+        ipLogMap[driver_info.node_ip_address],
+      )}?fileName=${type === "DRIVER" ? job_id : `driver-${submission_id}`}`;
+    }
+
+    if (link) {
+      return (
+        <Link to={link} target="_blank">
+          Log
+        </Link>
+      );
+    }
+
+    return "-";
```

Comment:
nit / personal pref:
```suggestion
    if (driver_agent_http_address) {
      link = `/log/${encodeURIComponent(
        `${driver_agent_http_address}/logs`,
      )}`;
    } else if (driver_info && ipLogMap[driver_info.node_ip_address]) {
      link = `/log/${encodeURIComponent(
        ipLogMap[driver_info.node_ip_address],
      )}`;
    } else {
      return "-";
    }

    link += `?fileName=${type === "DRIVER" ? job_id : `driver-${submission_id}`}`;

    
    return (
      <Link to={link} target="_blank">
        Log
      </Link>
    );
```